### PR TITLE
refactor all instances of amy and bob, update test cases

### PR DESCRIPTION
**Pull request by fatclarence**
Replaced AMY with CHICKEN_RICE
Replaced BOB with NASI_LEMAK
Update instances in test case with people names into food items names
Update JSON files with names of food items

### CODE REVIEW

File: src/test/java/seedu/savenus/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -28,35 +28,40 @@
...
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
-    public static final String OPENING_HOURS_DESC_AMY = " " + PREFIX_OPENING_HOURS + VALID_OPENING_HOURS_AMY;
-    public static final String OPENING_HOURS_DESC_BOB = " " + PREFIX_OPENING_HOURS + VALID_OPENING_HOURS_BOB;
-    public static final String RESTRICTIONS_DESC_AMY = " " + PREFIX_RESTRICTIONS + VALID_RESTRICTIONS_AMY;
-    public static final String RESTRICTIONS_DESC_BOB = " " + PREFIX_RESTRICTIONS + VALID_RESTRICTIONS_BOB;
+    public static final String VALID_NAME_CHICKEN_RICE = "Amy Bee";
+    public static final String VALID_NAME_NASI_LEMAK = "Bob Choo";
+    public static final String VALID_PRICE_CHICKEN_RICE = "11111111";
+    public static final String VALID_PRICE_NASI_LEMAK = "22222222";
```
Comment (robytanama) [inline src/test/java/seedu/savenus/logic/commands/CommandTestUtil.java]: Do you think we should change this to something more food appropriate? Just a suggestion it's not very important

File: src/test/java/seedu/savenus/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -84,34 +84,40 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + INVALID_TAG_DESC, Tag.MESSAGE_CONSTRAINTS); // invalid tag
 
         // invalid price followed by valid description
-        assertParseFailure(parser, "1" + INVALID_PRICE_DESC + DESCRIPTION_DESC_AMY, Price.MESSAGE_CONSTRAINTS);
+        assertParseFailure(parser, "1" + INVALID_PRICE_DESC
+                                                + DESCRIPTION_DESC_CHICKEN_RICE, Price.MESSAGE_CONSTRAINTS);
```
Comment (robytanama) [inline src/test/java/seedu/savenus/logic/parser/EditCommandParserTest.java]: maybe the indentation can be better?

Comment (robytanama) [reply src/test/java/seedu/savenus/logic/parser/EditCommandParserTest.java]: this goes for the whole of this file

File: src/test/java/seedu/savenus/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (fatclarence) [reply src/test/java/seedu/savenus/logic/commands/CommandTestUtil.java]: I overlooked this oh my goodness. I go change don't merge yet then !!

**Pull request merged**